mysql-test/suite/perfschema/t/bug91541.test

```diff
@@ -0,0 +1,75 @@
+--echo #
+--echo # Test for bug bug91541 ("Flush status" statement adds twice to global values)
+--echo #
+
+--source include/have_perfschema.inc
+
+SET @saved_show_compatibility_56 = @@global.show_compatibility_56;
+SET GLOBAL show_compatibility_56 = OFF;
+
+--echo #
+--echo # default connection
+--echo #
+--let $initial_global_value = query_get_value(SHOW GLOBAL STATUS LIKE 'Slow_queries', Value, 1)
+--let $default_session_value = query_get_value(SHOW SESSION STATUS LIKE 'Slow_queries', Value, 1)
+--let $assert_cond = $default_session_value = 0
+--let $assert_text = Default session value must be zero
+--source include/assert.inc
+
+--source include/count_sessions.inc
+
+--echo #
+--echo # first connection
+--echo #
+--connect (con1,localhost,root,,)
+--let $current_global_value = query_get_value(SHOW GLOBAL STATUS LIKE 'Slow_queries', Value, 1)
+--let $first_session_value = query_get_value(SHOW SESSION STATUS LIKE 'Slow_queries', Value, 1)
+--let $assert_cond = $current_global_value - $initial_global_value = 0
+--let $assert_text = Global value requested from the first session must not change
+--source include/assert.inc
+--let $assert_cond = $first_session_value = 0
```

Comment:
A brand new connection will start with zero session counter for sure?

Reply:
Agree. Assertion removed.